cli/tests/test_parallelize_command.rs

```diff
@@ -664,6 +664,115 @@ fn test_parallelize_complex_nonlinear_target() {
     ");
 }
 
+#[test]
+fn test_parallelize_immutable_base_commits() {
+    let test_env = TestEnvironment::default();
+    test_env.run_jj_in(".", ["git", "init", "repo"]).success();
+    let work_dir = test_env.work_dir("repo");
+
+    work_dir.run_jj(["new", "root()", "-m=x"]).success();
+    work_dir.run_jj(["new", "-m=x1"]).success();
+    work_dir.run_jj(["new", "-m=x2"]).success();
+    work_dir.run_jj(["new", "-m=x3"]).success();
+
+    work_dir.run_jj(["new", "root()", "-m=y"]).success();
+    work_dir.run_jj(["new", "-m=y1"]).success();
+    work_dir.run_jj(["new", "-m=y2"]).success();
+
+    work_dir
+        .run_jj([
+            "config",
+            "set",
+            "--repo",
+            "revset-aliases.'immutable_heads()'",
+            "description(exact:'x\n') | description(exact:'y\n')",
+        ])
+        .success();
+    work_dir
+        .run_jj(["config", "set", "--repo", "revsets.log", "all()"])
+        .success();
+    insta::assert_snapshot!(get_log_output(&work_dir), @r"
+    @  1a0f8336974a y2 parents: y1
+    ○  0e07ea90229f y1 parents: y
+    ◆  a0fb97fc193f y parents:
+    │ ○  d6c30fecfe88 x3 parents: x2
+    │ ○  6411b5818334 x2 parents: x1
+    │ ○  6d01ab1fb731 x1 parents: x
+    │ ◆  8ceb28e1dc31 x parents:
+    ├─╯
+    ◆  000000000000 parents:
+    [EOF]
+    ");
+
+    work_dir
+        .run_jj(["parallelize", "description(x)::", "description(y)::"])
```

Comment:
nit: since `description()` uses substring matching by default, you can drop the `::` operators